packages/connect-disputable-voting/src/models/Vote.ts

```diff
@@ -295,6 +297,16 @@ export default class Vote {
     )
   }
 
+  async formattedSettlementOffer(): Promise<string | null> {
+    if (!this.settlementOffer) {
+      return null
+    }
+
+    const collateralRequirement = await this.collateralRequirement()
+    const token = await collateralRequirement.token()
+    return formatBn(this.settlementOffer, token.decimals)
```

Comment:
Probably something to improve going forward but I think that doing two subgraph requests to get the `token.decimals` is going to become expensive when the graph has an indexer network.

Reply:
definitely, we can improve this